lib/Request.php

```diff
@@ -52,7 +52,10 @@ public static function request($request, $output_format = "xml", $decode_json =
             "Accept: ".$accept,
             "Authorization: Bearer ".$token
         );
-
+		/// enable GZIP compression for post MLSGRID Api
```

Comment:
Perhaps remove GRID from the comment? I think it's sufficient to say that it's for GZIP compression.